Gemfile.jruby-1.9.lock.release

```diff
@@ -219,9 +220,9 @@ GEM
       logstash-core-plugin-api (>= 1.60, <= 2.99)
       metriks
       thread_safe
-    logstash-filter-mutate (3.1.7)
+    logstash-filter-mutate (3.2.0)
```

Comment:
Support conversion of booleans to integers

Reply:
I would suggest to hold off this until the next minor release. It changes behavior and is arguable if the old behave was a bug or expected.